worlds/animal_well/options.py

```diff
@@ -180,6 +225,11 @@ class AnimalWellOptions(PerGameCommonOptions):
     bubble_jumping: BubbleJumping
     disc_hopping: DiscHopping
     wheel_tricks: WheelTricks
+    ball_throwing: BallThrowing
+    wheel_hopping: WheelHopping
```

Comment:
```suggestion
```
This is in here twice

Reply:
Byproduct of my line-by-line recovery of this file :P